buildSrc/src/main/kotlin/toolkit-kotlin-conventions.gradle.kts

```diff
@@ -1,6 +1,10 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import gradle.kotlin.dsl.accessors._a21abdf97e1bd2457f066e8d5acc3733.check
+import gradle.kotlin.dsl.accessors._a21abdf97e1bd2457f066e8d5acc3733.detekt
```

Comment:
what's with these imports

Reply:
IntelliJ being dumb. they arent needed

Reply:
It adds them if you use the quick fix on a fully qualified import